notes.txt

```diff
@@ -20,3 +20,28 @@ UI Elements the we need to wireframe -
 --------> When videos are played, they pop up as a centered overlay? 
 
 
+
```

Comment:
I love this style of pseudo coding out our wireframe for the MovieDetails component. Very easy to follow and implement!!